Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/LightingOptions.azsli

```diff
@@ -102,10 +102,6 @@ option bool o_enableIBL = true;
 option bool o_enableSubsurfaceScattering = false;
 option bool o_area_light_validation = false;
 
-#if ENABLE_CLEAR_COAT
```

Comment:
Why remove this? Dont we still want to protect the shader option or it may throw out a shader warning.